helm/panda/charts/jedi/sandbox/rucio.cfg

```diff
@@ -0,0 +1,22 @@
+[common]
```

Comment:
rucio.cfg and idds.cfg should be put under helm/panda/sandbox. They are copied to /data/panda/

https://github.com/PanDAWMS/panda-k8s/blob/main/helm/panda/charts/jedi/templates/deployment.yaml#L82

JEDI and server containers already have symlinks to those cfg files 

https://github.com/PanDAWMS/panda-server/blob/master/Dockerfile#L52-L53

So you don't have to tweak them in init-blah.

Reply:
ok. will do it.